mender.go

```diff
@@ -327,7 +330,14 @@ func (m *mender) loadAuth() menderError {
 }
 
 func (m *mender) IsAuthorized() bool {
-	return m.authMgr.IsAuthorized()
+	if m.authMgr.IsAuthorized() {
+		log.Info("authorization data present and valid")
```

Comment:
Maybe move to `Debug`? This will be hit 99% of the time for a healthy device, won't it?